CloudyCider/views/components/EC2/EC2List.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  EC2List.swift
+//  CloudyCider
+//
+//  Created by Matvii Hodovaniuk on 7/7/19.
+//  Copyright © 2019 Matvii Hodovaniuk. All rights reserved.
+//
+
+import SwiftUI
+
+struct EC2List: View {
+  @EnvironmentObject var store: EC2Store
```

Comment:
since this app would display not only EC2 entities, but entities from other services in the future, this really shouldn't be `@EnvironmentObject`, but `@ObjectBinding`. Since not all parts of the app would need access to this as an environment, it's better to pass this store explicitly.

Reply:
See [this article](https://www.hackingwithswift.com/quick-start/swiftui/whats-the-difference-between-objectbinding-state-and-environmentobject) for more details, environment is more suitable for passing state that's accessible in every part of the app and would be tedious to pass around explicitly, like dynamic type size, dark mode state, RTL state etc.